flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "publish-delay-hours": 1
+}
```

Comment:
Please drop it. We use it primarily for extra-data apps which are impossible to install unless updated.